packages/suite/src/components/suite/section/SectionItem.tsx

```diff
@@ -1,16 +1,9 @@
 import { HTMLAttributes, forwardRef } from 'react';
 
-import styled from 'styled-components';
 
 import { Flex, useMediaQuery, variables } from '@trezor/components';
 
 import { OutlineHighlight } from 'src/components/OutlineHighlight';
-import { SUBPAGE_NAV_HEIGHT } from 'src/constants/suite/layout';
-
-const Wrapper = styled.div`
-    /* height of secondary panel and a gap between sections */
-    scroll-margin-top: calc(${SUBPAGE_NAV_HEIGHT} + 79px);
```

Comment:
This was not working because we are not using real anchors from browser. Like using `id="labelling"` + urls like `/settings-index#labelling`. We are handling it on our own in `goto` and `useAnchor` functions. 

Maybe it's pity, but I didn't want to dig into the heavy refactoring just because of this.